src/Directory.Build.targets

```diff
@@ -20,7 +20,7 @@
 	<!-- Keep aligned with MSBuild extras' default ExtrasUwpMetaPackageVersion -->
 	<PackageReference Update="Uno.SourceGenerationTasks" Version="3.0.0" />
 	<PackageReference Update="Uno.SourceGeneration" Version="3.0.0" />
-	<PackageReference Update="Uno.Core" Version="2.2.0" />
+	<PackageReference Update="Uno.Core" Version="2.3.0-dev.3" />
```

Comment:
Do you need something specific in this version?

Reply:
Yes, I added the `RefCountDisposable` in there, it is used in the rentable streams 👍